app/helpers/notifier_helper.rb

```diff
@@ -58,12 +58,11 @@ def display_notes?(status, role, notes)
     (status == "submitted" || status == "request_amendment") && role == 'none' && !notes.empty?
   end
 
-  def determine_ssr(last_change, action_name)
+  def determine_ssr(last_change, action_name, deleted_ssrs)
     ssr_id = last_change.audited_changes['sub_service_request_id']
     if last_change.action == 'destroy'
       # This condition signifies a deleted SSR
       if SubServiceRequest.where(id: ssr_id).empty? && action_name == 'notify_user'
-        deleted_ssrs = @service_request.deleted_ssrs_since_previous_submission(true)
```

Comment:
Since adding delayed_job to emails, this method call wasn't capturing the deleted ssrs.  Instead we capture before putting into the delayed_job queue.  See next comment